docs/kb/semgrep-cloud-platform/missing-pr-commments.md

```diff
@@ -0,0 +1,50 @@
+---
+description: Use this reference to check why you may not be receiving Semgrep comments on PRs/MRs.
+tags:
+  - Semgrep CI
+  - PR Comments
+  - Rule Board
```

Comment:
```suggestion
  - Rule management
```
This feature's name is tricky. It's called Rule Management (capital M) in the header within the product, then "Rule" and "Board" in the left sidenav within the product, and it's a [T3 feature in product marketing docs](https://www.notion.so/semgrep/Product-and-feature-naming-conventions-16c6cf54cc05479ea85e1b10597a4147), which means that following convention, it should be "Rule management" and not "Rule Management" when it's being marketed. I apologize for the confusion.
<img width="942" alt="Snag_ac5649" src="https://github.com/returntocorp/semgrep-docs/assets/20766840/2a559440-5b37-45ed-9973-099c61bb57d2">